### ISSUE TITLE
Inconsistent results across devices

### ISSUE THREAD

**Issue by Paul-Hebert** [cloudfour/lighthouse-parade#82]
I got inconsistent results between two different computers. Both were Macbooks running Big Sur on the same network. 

Here's what I got on my newer M1 device:

<img width="748" alt="Screen Shot 2021-09-20 at 2 42 15 PM" src="https://user-images.githubusercontent.com/5798536/134081502-df51cfb5-fba1-44e5-b48c-e0ed25d5097b.png">

Here's the same site on my older non-M1 device:

![image](https://user-images.githubusercontent.com/5798536/134081569-84cba18f-89d3-4e93-81bb-31df6a26d640.png)

(Note this is also more consistent to the Lighthouse scores in the browser. The browser scores were consistent across devices)

The Lighthouse docs mention [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) but this seems pretty extreme.

I saw similar results on a number of sites. I tried making sure to use the same version of Node, reinstalling Chrome, restarting my machine, etc.

I'm not sure if this is related to it being an M1 machine or something else?

**Comment by spaceninja** [comment]
I was recently able to run Lighthouse Parade on my M1 and it seemed to run find. ¯\_(ツ)_/¯

**Comment by calebeby** [comment]
I re-ran this on my mac (not an M1) and asked Danielle run it on hers (M1) and we are still seeing this issue:

Non-M1:
<img width="816" alt="Screen Shot 2022-03-03 at 1 15 39 PM" src="https://user-images.githubusercontent.com/13206945/156654126-7a119c1e-8f8d-478d-9348-b950e57a8062.png">

M1:
<img width="816" alt="Screen Shot 2022-03-03 at 1 16 07 PM" src="https://user-images.githubusercontent.com/13206945/156654180-7c68026d-f194-4af4-9d32-91dc351cdc6f.png">

Command used:
```
nvm use 14
npx lighthouse-parade https://cloudfour.com/ --max-crawl-depth 3 --exclude-path-glob "/thinks/*" --exclude-path-glob "/presents/*"
```

**Comment by Paul-Hebert** [comment]
Thanks Caleb! That's interesting. I wonder why that is....

**Comment by calebeby** [comment]
I am looking into it more. I've looked through lighthouse issues to see if they list anything similar, but I couldn't find anything. Gonna try updating lighthouse too

**Comment by Paul-Hebert** [comment]
We were able to dig into this more in #92 

The issue crops up when "using virtualized x64 instead of natively running arm instructions" (quoting @calebeby because he has a better grasp of this than me)

I believe this means that if you're on an M1 machine you'll need to use Node 15 or 16 and confirm that your Node version is using `arm64` as the underlying architecture. @calebeby , @spaceninja , does that sound correct? If so what's the best place for us to document this? Should we add a Troubleshooting section to the Readme?

**Comment by spaceninja** [comment]
Yeah, adding a section in the README should be enough. I'd imagine something like:

---

# Mac M1 Note

Users running virtualized x64 versions of Node on Macs with M1 chips may see anomalous results. For best results on an M1 chip, ensure you are running the arm version of Node 15+.

When you run the command `node -p process.arch` you should see `arm64`. If you see `x86`, try uninstalling and reinstalling your version of Node, like so:

```
nvm uninstall 16
nvm install 16
nvm use 16
node -p process.arch
```

**Comment by calebeby** [comment]
Addressed in #93!!
